README.md

```diff
@@ -14,6 +14,42 @@ New code should adhere to the appropriate Google Style guide for the given langu
 * [HTML/CSS](https://google.github.io/styleguide/htmlcssguide.html)
 * [JavaScript](https://google.github.io/styleguide/jsguide.html)
 
+### Liquid
+
+#### Line length
+
+* Limit lines to a maximum of 80 columns in length.
+  * It is acceptable to exceed 80 columns if there is no way to break up the line, but developers should observe the 80 column limit where possible.
+
+#### Comments
+
+* Write comments in full sentences using standard Englisth capitalization and punctuation.
+
+##### Inline comments
```

Comment:
I like how you called out inline vs. block comments in the style guide here, however, it makes me question if these same rules apply to other liquid tags that have start/end tags, such as `capture`, `unless`, `raw`, etc.  Should we generalize this to Liquid tags in general?  Or do we need to add clarification for other tags?